crowdsec-docs/docs/console_management/intro.md

```diff
@@ -0,0 +1,38 @@
+---
+id: intro
+title: Introduction
+sidebar_position: 1
+---
+
+## Console Management
+
+The CrowdSec security engine is now able to receive instructions from the console.
+This is done via a polling API, it means that the CrowdSec Local API will use long polling to get orders from the CrowdSec Console.
+Currently, only 4 orders are available:
+ - Adding decisions from the console
+ - Delete decisions from the console
+ - Force pull the community-blocklist/third party list when an instance subscribe/unsubscribe to a blocklist in the Console
+ - Reauth to CAPI, when for example an instance is added to a tag
+
+
+## Enable console management
+
+The console management is still under feature flag on the CrowdSec Security Engine side. 
+There is two ways to enable this feature flag:
+ - Create or edit `/etc/crowdsec/feature.yaml` with the following flag
+
+```yaml
+- papi_client
+```
+ - Add this environement variable in `/lib/systemd/system/crowdsec.service`:
```

Comment:
I'd probably say to use `systemctl edit crowdsec.service`, this will prevent conflict when upgrading crowdsec (and will edit the correct file regardless of how the user installed crowdsec)